apps/emqx_bridge_oracle/test/emqx_bridge_oracle_SUITE.erl

```diff
@@ -553,6 +556,23 @@ t_probe_with_nested_tokens(Config) ->
         ?assertEqual({ok, connected}, emqx_resource_manager:health_check(ResourceId))
     ).
 
+t_probe_with_inconsistent_datatype(Config) ->
+    ResourceId = resource_id(Config),
+    reset_table(Config),
+    ?assertMatch(
+        {ok, _},
+        create_bridge(Config, #{
+            <<"sql">> => sql_insert_template_with_inconsistent_datatype()
+        })
+    ),
+    %% Since the connection process is async, we give it some time to
+    %% stabilize and avoid flakiness.
+    ?retry(
+        _Sleep = 1_000,
+        _Attempts = 20,
+        ?assertEqual({ok, connected}, emqx_resource_manager:health_check(ResourceId))
```

Comment:
The associated ticket shows that it's the probe API that returns a big error, but this is not the brigde probe API, as suggested by the test name.

Reply:
Ops. I'll fix that.